recipes/bam-readcount/build.sh

```
@@ -1,10 +1,10 @@
 #!/bin/bash -ex
 
-wget https://github.com/boostorg/boost/releases/download/boost-1.85.0/boost-1.85.0-cmake.tar.gz
+wget https://github.com/boostorg/boost/releases/download/boost-1.86.0/boost-1.86.0-cmake.tar.gz
 
-mv boost-1.85.0-cmake.tar.gz vendor/boost-1.55-bamrc.tar.gz
+mv boost-1.86.0-cmake.tar.gz vendor/boost-1.55-bamrc.tar.gz
 
-mkdir -p "${PREFIX}/bin"
+ln -sf $(which libtool) "${PREFIX}/bin/libtool"
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)